### ISSUE TITLE
The deck gets closed before I finish the cards on it.

### ISSUE THREAD

**Issue by felipecarvalho87** [ankidroid/Anki-Android#5302]
When I am reviewing the deck , sometimes the app goes back to the "home page" and says that i have no cards to review. 

If I close the app , and open again later the cards appears again..

Does anybody have that problem too? It started recently (no more than one month ago I think 

ps: I am new , sorry if i did not follow the guidelines for posting..

**Comment by mikehardy** [comment]
Hi there!

It would be helpful to know your AnkiDroid version, Android version, and device make/module

This sounds like you may have configured a time limit for reviews though, like setting the maximum time to review a day? Or the maximum number of reviews?

**Comment by felipecarvalho87** [comment]
Thanks for your help Mike!

Let's see :

AnkiDroid version 2.8,4

Android version : OxygenOS 9.0.5

Device : OnePlus 5t

I am not sure if I did this configuration... I just take a look the "Reviewing " menu and did not find this configuration..

In fact what I would like to do (but was not successful) is to set a max number of TOTAL cards that the APP shows me on a Day. If a skip a day or if there is just a lot of cards that need reviewing, the session takes too long .. 

I could only find this option regarding NEW cards, but not the TOTAL number..




Thanks a lot for your help.

**Comment by mikehardy** [comment]
I have a OnePlus 5 - nice phones :-). I'm still not sure on your original issue but your config sounds fine. Just in case, you might try the latest alpha versions - they are actually a little more stable than the 2.8.x series now, but we're waiting on a couple features to land before going beta. Also they are backwards/forwards compatible so no big deal if it doesn't work and you want to revert: https://github.com/ankidroid/Anki-Android/wiki/Installation#beta--alpha-releases

I doubt that will affect your original issue but it might

I believe the way to cap reviews is like so in deck options (available with a long-press on a deck in the deck list https://apps.ankiweb.net/docs/manual.html#reviews 

beware deck group settings are applied to the whole *deck group* - deck groups can contain 1 or more decks so you can limit things to a single deck or apply the same setting to a bunch of decks that you group). That wasn't so clear in the UI in 2.8.x but is more clear in 2.9.x if you go there

There is also a timebox limit you can search for - it may work better for you?

**Comment by timrae** [comment]
I just skim read this, but it sounds like expected behaviour due to sibling
cards getting automatically buried.

You can test this theory next time it happens by choosing "unbury cards"
from the menu (with three dots icon) on the deck list screen after a study
session. This menu item only appears when there are cards to unbury.

**Comment by Anthropos888** [comment]
I think the "deck gets closed before I finish the cards" is just a result of AnkiDroid showing "cards due today" during reviews and "cards due now" in the main window.

Normally my review session also ends with some remaining cards as "red" number, but 0 0 0 in the main window. That was quite confusing for me as well but actually it's a good way to quickly see how many cards are still "due today", even if the "due now" counter in the main window says "0".

**Comment by felipecarvalho87** [comment]
Thanks for all the help from you guys!

Unfortunately, it is still happening but it is not such a major problem... 

I don't know how to explained more clearly and I tried some suggestions above but did not work :

mikehardy - I did not change the version, because like I said before it is not a major concern.. But I looked at the link you put and I could change the max number of cards per day! Thank you!

timrae - Thanks for your suggestion. I did not find unbury cards so I think that is not the problem.. I saw a menu empty cards but when I click on it says there are no cards on this condition .

Anthropos888 - Not sure if I follow .. But the thing is that a few months back the problem that I explained here did not happen... 

Thanks again all for your time!
